http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -247,7 +247,7 @@ object AppRoutes {
       "labels"
     )(x =>
       (x.cloudContext,
-       x.cloudContext,
```

Comment:
cloudContext is an object, i don't think this will work

Reply:
it's a class that has asString....when it's CloudContext.Gcp, this is equivelant of the previous value we had for googleProject

Reply:
see similar usage https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala#L182